app/helpers/proxmox_vm_attrs_helper.rb

```diff
@@ -114,11 +114,11 @@ def additional_attrs(vms, param_scope, start_checked)
       start_after_create: vms.start_after_create,
       templated: vms.templated,
     }
-    vms_keys = [:pool, :start_after_create]
+    vms_keys = [:cpu_type, :nameserver, :searchdomain, :hostname]
     extra_attrs = ActiveSupport::HashWithIndifferentAccess.new
     attributes.each do |key, value|
       camel_key = key.to_s.include?('_') ? snake_to_camel(key.to_s).to_sym : key
-      nested_key = vms_keys.include?(key) ? key : "config_attributes[#{key}]"
+      nested_key = vms_keys.include?(key) ? "config_attributes[#{key}]" : key
```

Comment:
What is the purpose of this?

Reply:
Some attributes are part of config_attributes hash within vm_attrs hash and some are direct vm_attributes hash. This loop assign names to them accordingly.